config.tfvars

```diff
@@ -92,6 +92,35 @@ confluence_db_iops                 = 1000
 # WARNING: Collaborative editing can be only enabled if the `domain` variable is set.
 #confluence_enable_synchrony = true
 
+################################################################################
+# Bitbucket Settings
+################################################################################
+
+# Helm chart version of Bitbucket
+bitbucket_helm_chart_version = "1.2.0"
+
+# Bitbucket license
+# To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_` (i.e. `TF_VAR_bitbucket_license`) and keep the below line commented out
+# If storing license as plain-text is not a concern for this environment, feel free to uncomment the following line and supply the license here
+#
+#bitbucket_license = "<LICENSE KEY>"
+
```

Comment:
```suggestion

# Bitbucket system admin credentials
# WARNING: In case you are restoring an existing dataset (see the `dataset_url` property below), you will need to use credentials
# existing in the dataset to set this section. Otherwise any other value for the `bitbucket_admin_*` properties below are ignored.
# To avoid storing system admin password in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_` (i.e. `TF_VAR_bitbucket_admin_password`) and keep the below line commented out
# If storing password as plain-text is not a concern for this environment.

# To pre-seed the Bitbucket with the system admin information, feel free to uncomment the following settings and supply the system admin information:
#
#bitbucket_admin_username      = "<USERNAME>"
#bitbucket_admin_password      = "<PASSWORD>"
#bitbucket_admin_display_name  = "<DISPLAY NAME>"
#bitbucket_admin_email_address = "<EMAIL ADDRESS>"

```